src/platform/forms/save-in-progress/actions.js

```diff
@@ -265,49 +265,40 @@ export function fetchInProgressForm(
   // TODO: Migrations currently aren’t sent; they’re taken from `form` in the
   //  redux store, but form.migrations doesn’t exist (nor should it, really)
   return (dispatch, getState) => {
-    const trackingPrefix = getState().form.trackingPrefix;
+    const { trackingPrefix } = getState().form;
     const apiUrl = inProgressApi(formId);
 
     // Update UI while we’re waiting for the API
     dispatch(setFetchFormPending(prefill));
 
     // Query the api and return a promise (for navigation / error handling afterward)
-    return fetch(apiUrl, {
-      credentials: 'include',
-      headers: {
-        'Content-Type': 'application/json',
-        'X-Key-Inflection': 'camel',
-        'Source-App-Name': window.appName,
+    return apiRequest(
+      apiUrl,
+      {
+        credentials: 'include',
+        headers: {
+          'Content-Type': 'application/json',
+          'X-Key-Inflection': 'camel',
+          'Source-App-Name': window.appName,
+        },
       },
-    })
-      .then(res => {
-        if (res.ok) {
-          return res.json();
-        }
-
-        // Make me a switch?
-        let status = LOAD_STATUSES.failure;
-        if (res.status === 401) {
-          dispatch(logOut());
-          status = LOAD_STATUSES.noAuth;
-        } else if (res.status === 403) {
-          status = LOAD_STATUSES.forbidden;
-        } else if (res.status === 404) {
-          status = LOAD_STATUSES.notFound;
-        }
-        return Promise.reject(status);
```

Comment:
Are we sure that relocating this block down below inside of a `catch` will mean that it still gets run when the response wasn't a 2XX status?

Reply:
Ah, you know what. Two things are occurring to me: 
1. I'm not certain whether my changes are going to handle 403s and 401s correctly.
2. Prior to my changes, I don't know understand how (or whether) 404s were being handled.

I'll have a closer look. If you have any thoughts off-hand, please let me know. Thanks!